build-system/tasks/extension-helpers.js

```diff
@@ -490,7 +490,7 @@ async function buildExtension(
     }
   }
 
-  await compileJison(path.join(extDir, '**', '*.jison'));
+  await compileJison(`${extDir}/**/*.jison`);
```

Comment:
Was this the problem line? Globby requires a unix-formatted path?

Reply:
Yep. See https://github.com/sindresorhus/globby/issues/130.